src/Bridges/Constraint/bridges/complex_norm_infinity.jl

```diff
@@ -0,0 +1,146 @@
+# Copyright (c) 2017: Miles Lubin and contributors
+# Copyright (c) 2017: Google Inc.
+#
+# Use of this source code is governed by an MIT-style license that can be found
+# in the LICENSE.md file or at https://opensource.org/licenses/MIT.
+
+"""
+    ComplexNormInfinityToSecondOrderConeBridge{T} <: Bridges.Constraint.AbstractBridge
+
+`ComplexNormInfinityToSecondOrderConeBridge` implements the following
+reformulation:
+
+  * ``(t, x) \\in NormInfinity(1+d)`` into ``(t, real(x_i), imag(x_i)) \\in SecondOrderCone()``
+    for all ``i``.
+
+## Source node
+
+`ComplexNormInfinityToSecondOrderConeBridge` supports:
+
+  * [`MOI.VectorAffineFunction{Complex{T}}`](@ref) in [`MOI.NormInfinityCone`](@ref)
+
+## Target nodes
+
+`ComplexNormInfinityToSecondOrderConeBridge` creates:
+
+  * [`MOI.VectorAffineFunction{T}`](@ref) in [`MOI.SecondOrderCone`](@ref)
+"""
+struct ComplexNormInfinityToSecondOrderConeBridge{T} <: AbstractBridge
+    ci::Vector{
+        MOI.ConstraintIndex{MOI.VectorAffineFunction{T},MOI.SecondOrderCone},
+    }
+end
+
+const ComplexNormInfinityToSecondOrderCone{T,OT<:MOI.ModelLike} =
+    SingleBridgeOptimizer{ComplexNormInfinityToSecondOrderConeBridge{T},OT}
+
+function bridge_constraint(
+    ::Type{ComplexNormInfinityToSecondOrderConeBridge{T}},
+    model::MOI.ModelLike,
+    f::MOI.VectorAffineFunction{Complex{T}},
+    s::MOI.NormInfinityCone,
+) where {T}
+    fi_s = MOI.Utilities.scalarize(f)
+    t = real(fi_s[1])
```

Comment:
What do we do with the imaginary part of `fi_s[1]` ? We should error if it's not zero

Reply:
Done